lib/bot.js

```diff
@@ -185,8 +187,16 @@ class Bot {
         return;
       }
 
+      const currentChannelUsernames = slackChannel.members.map( member =>
+        slackChannel._client.users[member].name
+      );
+
+      currentChannelUsernames.forEach( u => {
```

Comment:
This might be unwarranted paranoia, but I wonder if performance would be an issue for extremely active community Slacks here (with thousands of members). It's probably not a problem, but might be worth keeping in mind.